### ISSUE TITLE
k interval offset larger than 2?

### ISSUE THREAD

**Issue by rheacangeo** [GridTools/gt4py#48]
I have a stencil where the code specification indicates the need for a `with interval(3, 4)`, but this causes an error.
E.g. :

```python
@gtscript.stencil(backend="numpy")
def example(ub:  gtscript.Field[_dtype]):
 with computation(PARALLEL), interval(3, 4):
        ub = 5.
```
This raises and error indicating this interval is invalid, which looks like is stemming from the default `offset_limit` value in the make_axis_interval function:
```python
def make_axis_interval(bounds: tuple, *, offset_limit: int = 2)
 assert isinstance(bounds[0], (VarRef, UnaryOpExpr, BinOpExpr)) or (
          isinstance(bounds[0], int) and abs(bounds[0]) <= offset_limit
        )
       AssertionError
```
Is there a way to override the `offset_limit` from the frontend?

**Comment by egparedes** [comment]
This issue is basically a duplicate of #93 with an actual example

**Comment by tehrengruber** [comment]
This has been fixed in PR #176
